test/tasks/karpenter-test-suite.yaml

```
@@ -0,0 +1,127 @@
+# ginkgo-suite is a generic task that executes ginkgo on the given repository and ginkgo suite.
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: karpenter-test-suite
+  namespace: karpenter-tests
+spec:
+  description: |
+    Execute a karpenter test suite
+  params:
+    - name: cluster-name
+      default: $(context.taskRun.name)
+    - name: repository
+      default: github.com/aws/karpenter
+    - name: tag
+      default: v0.13.2
+    - name: command
+      default: go test ./pkg/...
```

Should this be the following?
```suggestion
      default: go test ./test/pkg/...
```